dateparser_scripts/write_complete_data.py

```diff
@@ -31,7 +31,7 @@ def _modify_relative_data(relative_data):
     modified_relative_data = OrderedDict()
     for key, value in relative_data.items():
         for i, string in enumerate(value):
-            string = RELATIVE_PATTERN.sub(r'(\\d+)', string)
+            string = RELATIVE_PATTERN.sub(r'(\\d+[.,]?\\d*)', string)
```

Comment:
3. Update the data converter script to include decimal places in regexes

Reply:
While we wait for a 2nd maintainer approval, I wonder if we should go for a more strict regular expression here. Something like `(\d+(?:[.,]\d+)?` maybe?

Reply:
@Gallaecio I've spent hours trying more specific regexes but for some reason, any regex I try besides the current one ends up with hundreds of failed tests and strings not translating. Feel free to try some things out and see if you can get it to work, but I think sticking to this regex that works is easiest.